tests/testsuite/fix.rs

```diff
@@ -805,7 +805,7 @@ fn does_not_warn_about_dirty_ignored_files() {
 }
 
 #[cargo_test]
-fn fix_all_targets_by_default() {
+fn do_not_fix_tests() {
```

Comment:
`_by_default`?  The name sounds like we should never fix tests

Reply:
Fixed!